mautic/mautic
# Update datafixture csv for page and email bundles

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [x]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #11471 



#### Description:

In the Doctrine fixtures, 2 emails and 1 landingspage is created. Unfortunately, their data still relies on the old builder. Possibly very old builder, as even disabling GrapesJS does not fix this.

When editing these pages or emails, clicking "build" brings up the old builder UI but it is entirely broken. The console is full of errors.

I suggest updating the csv files to create this content using the new structure.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Try to edit an email or landingspage. It should now work.
